compiler/damlc/BUILD.bazel

```diff
@@ -57,6 +57,7 @@ package_app(
     binary = ":damlc",
     resources = [
         ":ghc-pkg-dist",
+        "//compiler/damlc/daml-ide-core:hlint-yaml",
```

Comment:
This only matters for damlc-dist and the SDK tarball. To make sure that it is found in tests, you also need to add it to the `data` field of the `damlc` `da_haskell_binary` above.

Reply:
Ooh. Ok, I'll fix this.

Reply:
Just to be clear: That is (at least one of) the reasons why the lsp tests are failing.

Reply:
Yes, one of. It's not the whole story because tests are continuing to fail.